### ISSUE TITLE
Could you please provide the internal format of the file ‘news_summary.csv’?

### ISSUE THREAD

**Issue by wangyuxuan11** [jianmohqr/t5_small_finetune#1]

**Comment by qirui-huang** [comment]
You can download the file from this address: https://raw.githubusercontent.com/Shivanandroy/T5-Finetuning-PyTorch/main/data/news_summary.csv, and then use function pd.read_csv() to get the format.

**Comment by wangyuxuan11** [comment]
Thanks! Can this repo be used to train Chinese data?

**Comment by qirui-huang** [comment]
Yes, this is standard data format used for T5, which is usually used for translation tasks including Chinese-xxx languages.

**Comment by wangyuxuan11** [comment]
Thanks again for your code, but when I was training with my Chinese data set, I didn't seem to get the right results. Below is a screenshot of the ‘predictions.csv’ saved in the validation phase, it only seems to encode punctuation marks and English parts, do you know why?
<img width="405" alt="image" src="https://github.com/jianmohqr/t5_small_finetune/assets/111549316/1eca1cb2-f660-4f31-9b7b-a16be7b5f39a">
